### DISCUSSION TITLE
Can I host Vaultwarden on a ARM Server?

### DISCUSSION THREAD

**Discussion by DoYouKnowWhoElse** [dani-garcia/vaultwarden#4565] [General]
Can I host Vaultwarden on a ARM Server?

**Comment by BlackDex** [comment]
Yes

**Comment by samuellhwarren** [comment]
Yes, I do currently. it works well

**Comment by samuellhwarren** [comment]
https://github.com/samuellhwarren/docker-compose-vaultwarden if you want the docker-compose file..

just replace <text> placeholders with your info and you can run it just fine. My file has the admin side as well... you can link it up with something like sendgrid for smtp mail.
